newsletter-november-2022.rst

```diff
@@ -0,0 +1,97 @@
+
+
+Possible issues
+---------------
+
+If you find regressions in any new releases of the `sphinx-rtd-theme <https://sphinx-rtd-theme.readthedocs.io/>`_,
+please don't hesitate to `open an issue on GitHub <https://github.com/readthedocs/sphinx_rtd_theme/>`_.
+
+We continue planning to be more active in deprecating old and outdated approaches to using our platform in Q4.
+We don't have anything firm to announce here yet,
+but we do plan to be more active in removing these features in the coming months.
+
+
+Tip of the month
+----------------
+
+This Tip of the Month comes from our own experience. We have greatly benefitted from the following two Python/Django projects:
+
+- `blacken-docs <https://github.com/adamchainz/blacken-docs>`_ is a pre-commit linter that checks and formats your Python code. In other words, it helps you make sure that your documentation's code examples look great.
+- `django-upgrade <https://github.com/adamchainz/django-upgrade>`_ saves us a lot of time each time we upgrade to the latest Django version.
+
+As you might have noticed, both of these projects are maintained by `Adam Johnson <https://adamj.eu/>`_. Thanks, Adam 👋
+
+
+Awesome Project of the month
+----------------------------
+
+`As we also tweeted <https://twitter.com/readthedocs/status/1581949857865965569>`_, we are really big fans of how the `Wagtail <https://wagtail.org/>`_ community has built its documentation on `docs.wagtail.org <https://docs.wagtail.org/>`_. Our favorite parts are...
+
+Custom theme 🎨:
+  Wagtail uses a beautiful custom Sphinx theme with dark mode support
+
+Release notes 🚢:
+  Its documentation includes carefully curated 💅​ release notes that developers are happy to read. They include what’s new, bug fixes, new features and upgrade notes between versions. Take a look at their `latest major release notes <https://docs.wagtail.org/en/latest/releases/4.0.html>`_.
+
+Contribution guide 👩‍👩‍👧​👨‍👨‍👦‍👦​:
+  Wondering how to write a GOOD “Contribution guide” 👩‍👩‍👧​👨‍👨‍👦‍👦​👨‍👩‍👧‍👦​​? Take a look at Wagtail's `Contribution Guide <https://docs.wagtail.org/en/latest/contributing/index.html>`_. It gives a brief overview of how you can contribute and once you have decided what you want to do, you can read more in 11 carefully crafted categories.
+
+Personas 👩🏽‍💻​:
+  Wagtail divides the readers in two categories: “developers 👩🏽‍💻​ that want to install and maintain their Wagtail instance” and “users👨‍💼 of a Wagtail-powered site”. This makes it easier for each of these two personas to find what they are looking for immediately.
+
+5 out of 6 contributors write documentation 🎉 🎉 🎉:
+  Wagtail has a stunning amount of documentation contributors! Out of the ~600 contributors to Wagtail, ~500 of those have written documentation. Most of them added changelog entries or release notes, since adding code changes requires updating the changelog. This is a great way to potentially turn your code contributors into documentation contributors.
+
+Awesome List 🕶️
```

Comment:
```suggestion
Awesome Read the Docs Projects List 🕶️
```

Reply:
Agreed being more explicit here. In the future this could be part of our "sign off" instead of a full section, but I think it's fine for now.